admin_guide/high_availability.adoc

```diff
@@ -33,9 +33,9 @@ unavailable or if the service that *Keepalived* is watching does not respond,
 the VIP is switched to another host from the set. Thus, a VIP is always serviced
 as long as a host is available.
 
-When a host running *Keepalived* passes the *check* script it becomes available to
-become the *MASTER* based on its priority and the priority of the current *MASTER*.
-The configurable xref:vrrp-preemption[preemption strategy] determines what to do.
+When a host running *Keepalived* passes the *check* script, the host can become
+the *MASTER* based on its priority and the priority of the current *MASTER*, as
+determined by the xref:vrrp-preemption[preemption strategy].
```

Comment:
Why *MASTER*? This formatting is not used further down in a similar context:
"This enables any of the ipfailover nodes to become the master..."